Library/Formula/open-mesh.rb

```diff
@@ -16,11 +16,17 @@ class OpenMesh < Formula
   head "http://openmesh.org/svnrepo/OpenMesh/trunk/", :using => :svn
 
   depends_on "cmake" => :build
-  depends_on "qt"
+  depends_on "qt" => :optional
```

Comment:
@DomT4 So what's the recommended usage of `:optional` and `if build.with?` ? I'm often confused when using things like this.